### PR TITLE
Comment that migration_lock.toml should be put in Git

### PR DESCRIPTION
**Pull request by Jolg42**

### CODE REVIEW

File: migration-engine/connectors/migration-connector/src/migrations_directory.rs [hunk 1]
```diff
@@ -64,6 +64,7 @@ pub fn write_migration_lock_file(migrations_directory_path: &str, provider: &str
     let mut file = std::fs::File::create(&file_path)?;
     let content = format!(
         r##"# Please do not edit this file manually
+# It should be added in your version-control system (i.e. Git)
```
Comment (tomhoule) [inline migration-engine/connectors/migration-connector/src/migrations_directory.rs:67]: super nitpicky, but "e.g." would be more accurate than "i.e" there.

**Review by tomhoule — APPROVED** [review]
Looks good!

**Comment by tomhoule** [comment]
Let's merge this!

File: migration-engine/connectors/migration-connector/src/migrations_directory.rs (hunk 1 above)
Comment (Jolg42) [reply migration-engine/connectors/migration-connector/src/migrations_directory.rs:67]: Oh right

Comment (Jolg42) [reply migration-engine/connectors/migration-connector/src/migrations_directory.rs:67]: I guess it's ok like this 😄

**Pull request merged**